style.css

```diff
@@ -110,3 +107,170 @@ a {
   cursor: pointer;
   color: #7f8cff;
 }
+
+/* Portfolio Styles
+************************** */
+.portfolio {
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+  row-gap: 100px;
+  margin: 100px 0;
+}
+
+.card {
+  background-color: white;
+  border: 1px solid #dfe1e6;
+  width: 90%;
+  margin: 0 auto;
+  border-radius: 16px;
+}
+
+.card-container {
+  padding: 15px;
+  display: flex;
+  flex-direction: column;
+}
+
+.card img {
+  border-radius: 16px;
+}
+
+h3 {
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  margin: 8px 0;
+}
+
+h4 {
+  color: #7a869a;
+  margin: 0;
+}
+
+h4 span {
+  color: #344563;
+}
+
+.card p {
+  color: #344563;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.card ul,
+.about ul {
+  padding: 0;
+}
+
+.card li {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  color: #6070ff;
+  display: float;
+  float: left;
+  list-style-type: none;
+  margin: 0 15px 0 0;
+  background-color: #ebebff;
+  border-radius: 8px;
+  padding: 5px 10px;
+}
+
+button {
+  color: #396df2;
+  background-color: white;
+  border: 1px solid #6070ff;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border-radius: 8px;
+  align-self: flex-start;
+  padding: 8px 8px;
+  margin-top: 1rem;
+}
+
+form button {
+  margin-bottom: 20px;
+}
+
+button:hover {
+  background: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  color: white;
+}
```

Comment:
[OPTIONAL]

Consider changing the color of the lines too, that way the hamburger button will also be visible on hover :mag: